wsagent/che-datasource-api/src/main/java/org/eclipse/che/datasource/api/ssl/TrustStoreObject.java

```diff
@@ -0,0 +1,61 @@
+package org.eclipse.che.datasource.api.ssl;
+
+import com.google.inject.Inject;
+import com.google.inject.name.Named;
+import org.apache.commons.fileupload.FileItem;
+
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.security.cert.Certificate;
+import java.security.cert.CertificateFactory;
+import java.util.Iterator;
+
+/**
+ * Created by test on 7/15/17.
+ */
+public class TrustStoreObject extends KeyStoreObject{
+
+    public TrustStoreObject() throws Exception {
+        super("http://192.168.1.35:8080");
+    }
+
+
+    @Override
+    protected String getKeyStorePassword() {
+        return SslKeyStoreService.getDefaultTrustorePassword();
+    }
+
+    @Override
+    protected String getKeyStorePreferenceName() {
+        return TRUST_STORE_PREF_ID;
+    }
+
+    @Override
+    public Response addNewKeyCertificateAndRespond(@QueryParam("alias") String alias,
+                                                   Iterator<FileItem> uploadedFilesIterator) throws Exception {
+        addNewServerCACert(alias, uploadedFilesIterator);
+        return Response.ok("", MediaType.TEXT_HTML).build();
```

Comment:
I guess it would be much better if you could provide some meaningful instead of `""`, don't you agree?